src/Avalonia.Controls/Grid.cs

```diff
@@ -164,41 +164,43 @@ public bool ShowGridLines
         /// <summary>
         /// Returns a ColumnDefinitions of column definitions.
         /// </summary>
+        [MemberNotNull(nameof(_extData))]
```

Comment:
Does this attribute work here? I thought it makes sense only when set on specific get and set methods inside.

Reply:
Yes, it has the same effect as putting the attribute on both the get and set methods.